app/views/miq_ae_customization/_dialog_sample.html.haml

```diff
@@ -42,7 +42,8 @@
                         - date_val = field.value ? field.value.split(" ") : ["#{t.month}/#{t.day}/#{t.year}"]
                         = datepicker_input_tag("miq_date_1", h(date_val[0]),
                           :class    => "css1",
-                          :readonly => "true")
+                          :readonly => "true",
+                          :data_date_start => Time.zone.now.strftime("%Y, %m, %d"))
```

Comment:
This would disallow selecting a date in the past for all datepickers, not just the ones which don't allow selecting
a date in the past, correct?